CDCgov/prime-reportstream
# 15523 - added to the join to help speed up the query.

This PR ...

Bug Fix slow load times for some receivers in Daily Data API call load time on Staging and Production

Test Steps:
1. Login to staging
2. Dupe as 'Ignore' org
3. Navigate to Daily Data
4. Check that load time is less than 10 seconds (was taking 30)

## Changes
- added to join
```
     and "public"."action"."receiving_org" = "public"."report_file"."receiving_org"
     and "public"."action"."receiving_org_svc" = "public"."report_file"."receiving_org_svc"
```

## Checklist

### Testing


## Linked Issues
- Fixes #15523
